app/lib/actions/katello/repository/clone_to_environment.rb

```diff
@@ -13,7 +13,8 @@ def plan(repository, environment)
               plan_action(Repository::Create, clone, true, false)
             else
               #only clear if it should be empty, but its not
-              plan_optional_pulp_action([Actions::Pulp::Repository::Clear], clone, SmartProxy.pulp_master) if (!clone.yum? || !clone.empty_in_pulp?)
+              plan_optional_pulp_action([Actions::Pulp::Repository::Clear], clone, SmartProxy.pulp_master) unless SmartProxy.pulp_master.pulp3_support?(clone)
```

Comment:
plan_optional_pulp_action should ignore it if its pulp3, so i don't think we need that unless check there

Reply:
right, missed that.